cclib/parser/orcaparser.py

```diff
@@ -377,6 +377,56 @@ def splitter(line):
             self.is_DFT = True
             # In theory we could also parse the functional from this section, 
             # but sadly ORCA doesn't print simple functional names.
+        
+        # --------------------
+        # CPCM SOLVATION MODEL
+        # --------------------
+        # CPCM parameters:
+        #   Epsilon                                         ...       2.3741
+        #   Refrac                                          ...       1.4970
+        #   Rsolv                                           ...       1.3000
+        #   Surface type                                    ... GAUSSIAN VDW
+        #   Epsilon function type                           ...         CPCM
+        # Radii:
+        #  Radius for O  used is    3.4469 Bohr (=   1.8240 Ang.)
+        #  Radius for H  used is    2.4944 Bohr (=   1.3200 Ang.)
+        # Calculating surface                               ...        done! (  0.0s)
+        # GEPOL surface points                              ...          244
+        # GEPOL Volume                                      ...     194.1477
+        # GEPOL Surface-area                                ...     165.6341
+        # Calculating surface distance matrix               ...        done! (  0.0s)
+        # Performing Cholesky decomposition & store         ...        done! (  0.0s)
+        # Overall time for CPCM initialization              ...                 0.0s
+        if line.strip() == "CPCM SOLVATION MODEL":
+            # We can assume we're using CPCM if we see this line (although confusingly
+            # SMD uses it too, we can correct later).
```

Comment:
SMD can be based on C-PCM or IEF-PCM (the original definition) for the electrostatic contribution.

Reply:
Ah ok, I'll modify the solvent model to differentiate between the two. From my reading ORCA user C-PCM while Gaussian does the IEF-PCM